.travis.yml

```diff
@@ -1,19 +1,20 @@
 language: php
 
 php:
-    - 5.4
-    - 5.5
```

Comment:
this should be dropped only if the composer.json removes them. Otherwise, you should force to run them on trusty instead (which you can do by specifying them in the matrix config to configuring `dist` for these jobs)